src/polyswarm_api/__init__.py

```diff
@@ -542,16 +557,14 @@ def _fix_result(self, result):
 
         return await self.scan_files(file_list)
 
-    async def search_hashes(self, hashes, hash_type='sha256'):
+    async def search_hashes(self, hashes):
```

Comment:
we want to force the user to specify the type of hash. And if they do not specify the type of hash then we will assume sha256. So, I need you to re-add the "hash_type" arg to all of these functions. Then update the docs to tell the user to specify the hash type on the command line. And if no hash type is provided, then we will assume sha256.

Reply:
For now, lets autodetect. It's an easy change later (since we aren't changing the backend) and it makes it easier for the user.